ui/src/admin/components/chronograf/OrgTableRow.js

```diff
@@ -0,0 +1,92 @@
+import React, {PropTypes} from 'react'
+
+import Authorized, {SUPERADMIN_ROLE} from 'src/auth/Authorized'
+
+import Dropdown from 'shared/components/Dropdown'
+
+import {USER_ROLES} from 'src/admin/constants/dummyUsers'
+import {USERS_TABLE} from 'src/admin/constants/chronografTableSizing'
+
+const OrgTableRow = ({
+  user,
+  organization,
+  onToggleUserSelected,
+  selectedUsers,
+  isSameUser,
+  onChangeUserRole,
+}) => {
+  const {colOrg, colRole, colSuperAdmin, colProvider, colScheme} = USERS_TABLE
+
+  const isSelected = selectedUsers.find(u => isSameUser(user, u))
+
+  const currentRole = user.roles.find(
+    role => role.organization === organization.id
+  )
+
+  return (
+    <tr className={isSelected ? 'selected' : null}>
+      <td
+        onClick={onToggleUserSelected(user)}
+        className="chronograf-admin-table--check-col chronograf-admin-table--selectable"
+      >
+        <div className="user-checkbox" />
+      </td>
+      <td
+        onClick={onToggleUserSelected(user)}
+        className="chronograf-admin-table--selectable"
+      >
+        <strong>
+          {user.name}
+        </strong>
+      </td>
+      <td style={{width: colOrg}}>
+        <span className="chronograf-user--org">
+          {organization.name}
+        </span>
+      </td>
+      <td style={{width: colRole}}>
+        <span className="chronograf-user--role">
+          <Dropdown
+            items={USER_ROLES.map(r => ({
```

Comment:
This should be performed alongside the other variable declarations above

Reply:
👍